libs/libzauth/libzauth-c/Makefile

```diff
@@ -19,7 +21,7 @@ all: build
 clean:
 	cargo clean
 	rm -rf test/target
-	rm -rf deb$(PREFIX)/
```

Comment:
don't we want to clean both here?

Reply:
cleaning an install (which is what prefix_install is for) would be performing an uninstall. the install rule places results directly on the filesystem, not in a temporary location, so it does not need cleaned up after.